### PR TITLE
Feat/support duration rule

### PR DESCRIPTION
**Pull request by PeterCervienka**

### CODE REVIEW

File: src/utils/helpers.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { isNil } from 'lodash';
```
Comment (PeterCervienka) [inline src/utils/helpers.js]: the name is not the most original name on the world, but my imagination got me nothing better :) feel free to come with better naming ;)

Comment (relekang) [inline src/utils/helpers.js]: Calling it helpers inside utils is like `helpers/helpers`. What about `utils/numbers.js`?

File: src/rules/duration.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { get, isObject } from 'lodash';
```
Comment (relekang) [inline src/rules/duration.js]: Please add `// @flow` to this file.

File: src/utils/__tests__/helpers_tests.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+/* eslint-env jest */
```
Comment (relekang) [inline src/utils/__tests__/helpers_tests.js]: Please add `// @flow` to this file.

File: src/utils/helpers.js (hunk 1 above)
Comment (relekang) [inline src/utils/helpers.js]: Please add `// @flow` to this file.

Comment (PeterCervienka) [inline src/utils/helpers.js]: done :)

File: src/rules/duration.js (hunk 2 above)
Comment (PeterCervienka) [inline src/rules/duration.js]: done :)

File: src/utils/__tests__/helpers_tests.js (hunk 3 above)
Comment (PeterCervienka) [inline src/utils/__tests__/helpers_tests.js]: done :)

**Pull request merged**